### PR TITLE
[ci]: build vs image in official build

### PR DESCRIPTION
**Pull request by lguohan**
Signed-off-by: Guohan Lu <lguohan@gmail.com>

#### Why I did it
build vs image in official build

#### How I did it
- use template for vs image

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012

### CODE REVIEW

File: .azure-pipelines/build-template.yml [hunk 1]
```diff
@@ -82,20 +84,31 @@ jobs:
...
-      fi
-      if [ ${{ parameters.raw_image }} == true ];then
-        make USERNAME=admin $CACHE_OPTIONS SONIC_BUILD_JOBS=$(nproc) target/sonic-${{ parameters.platform }}.raw
-      fi
-      if [ ${{ parameters.sync_rpc_image }} == true ];then
-        make USERNAME=admin $CACHE_OPTIONS SONIC_BUILD_JOBS=$(nproc) ENABLE_SYNCD_RPC=y target/docker-syncd-${{ parameters.platform_short }}-rpc.gz
+
+      if [ ${{ parameters.platform }} == vs ]; then
+        if [ ${{ parameters.dbg_image }} == true ]; then
+          make USERNAME=admin $CACHE_OPTIONS SONIC_BUILD_JOBS=$(nproc) INSTALL_DEBUG_TOOLS=y target/sonic-vs.img.gz && \
```
Comment (qiluo-msft) [inline .azure-pipelines/build-template.yml:90]: > && \ [](start = 115, length = 5)

Remove them and just use 2 lines?

**Comment by qiluo-msft** [comment]
>   - sonicbld_8c

What does `8c` mean?

---
Refers to: .azure-pipelines/build-template.yml:39 in 7342fd1. [](commit_id = 7342fd17d8e25e4b3788413e718555236a50c681, deletion_comment = False)

**Comment by lguohan** [comment]
8c-> 8 core

File: .azure-pipelines/build-template.yml (hunk 1 above)
Comment (lguohan) [reply .azure-pipelines/build-template.yml:90]: only when build successfull will do the mv

**Review by qiluo-msft — APPROVED** [review]

File: .azure-pipelines/build-template.yml (hunk 1 above)
Comment (qiluo-msft) [reply .azure-pipelines/build-template.yml:90]: there is `set -e`

Comment (lguohan) [reply .azure-pipelines/build-template.yml:90]: will look at it later.

**Pull request merged**